.github/workflows/tests_and_qc.yml

```diff
@@ -0,0 +1,57 @@
+# Built from:
+# https://docs.github.com/en/actions/guides/building-and-testing-python
+
+name: Tests and QC
+
+on:
```

Comment:
@Sigfried @hugwuoke I just started this pull request to run GitHub actions.
@hugwuoke I just want you to be aware of this but you don't need to know anything deep about how it works probably!

I'm setting it to run on:
- `workflow_dispatch`: This is what enables it to show up in the "Actions" menu for us to run manually
- `schedule`: Daily. Currently deactivated until I get the tests to pass.
- `push`: If we push directly to `develop` or `main`, it will trigger a run.
- `pull_request`: It's pretty cool! At the bottom of the PR page, you'll see a little section where it says whether it is running / has run a test, and you can click it to open the action log. Scroll down and take a look!